src/components/Card/Card.vue

```diff
@@ -41,6 +41,19 @@ export default {
     humidity: Number,
     timeStamp: String,
     weatherIconSrc: String
+  },
+  methods: {
+    formatWeatherIcon () {
+      const weatherIcons = ['01d', '01n', '02d', '02n', '03d', '04d', '09d', '10d', '10n']
+      if (weatherIcons.includes(this.weatherIconSrc)) {
+        return this.weatherIconSrc
+      } else if (!weatherIcons.includes(this.weatherIconSrc)) {
+        const nums = weatherIcons.find(icon => {
+          return this.weatherIconSrc.slice(0, 2) === icon.slice(0, 2)
+        })
+        return nums
```

Comment:
I can't remember if we made it into an issue or not, but we'll want to remember to add other conditionals just to make sure that some random icon number (i.e. 08d) is able to be handled in a way that won't break

Reply:
Yep- I just created an issue for this!

Reply:
After some of the checkins this morning, I'm curious if we'd even want to reconsider creating custom icons to cover the range of numbers referenced by the API data